.packit.yaml

```diff
@@ -157,12 +167,17 @@ jobs:
       distros: [RHEL-7.9-rhui]
   identifier: tests-7to8-aws-e2e
   tmt_plan: "^(?!.*upgrade_plugin)(?!.*tier[2-3].*)(?!.*rhsm)(?!.*c2r)(?!.*sap)(?!.*8to9)(.*e2e)"
-  tf_post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; echo 42; yum-config-manager --enable rhel-7-server-rhui-optional-rpms"
   tf_extra_params:
     environments:
       - tmt:
           context:
             distro: "rhel-7.9"
+        # tag resources as sst_upgrades to enable cost metrics collection
+        settings:
+          provisioning:
+            post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys; echo 42; yum-config-manager --enable rhel-7-server-rhui-optional-rpms"
```

Comment:
@fernflower  why the `echo 42`? :) is it forgotten debug cmd or it has actually some additional value?

Reply:
We can safely remove that now I guess :) I will fix it in the next patch when I will be squashing 2 last commits anyway, thanks